tooling/nargo_cli/tests/execution_success/simple_bitwise/src/main.nr

```diff
@@ -5,5 +5,6 @@ fn main(a: bool, b: bool, c: u8, d: u8) -> pub u8 {
     let x = c & d;
     let y = c ^ d;
     let z = c | d;
-    (i as u8) + (j as u8) + (k as u8) + x + y + z
+    (
+    i as u8 + j as u8 + k as u8 + x + y + z
```

Comment:
Not sure I like the removal of parentheses here. Also one is getting stranded on the previous line.

Reply:
I agree this is broken imo. The stranded paren just adds confusion.